Allure.Net.Commons/AllureConstants.cs

```diff
@@ -10,5 +10,11 @@ public sealed class AllureConstants
         public const string TEST_RESULT_CONTAINER_FILE_SUFFIX = "-container.json";
         public static string TEST_RUN_FILE_SUFFIX = "-testrun.json";
         public const string ATTACHMENT_FILE_SUFFIX = "-attachment";
+
+        public const string OLD_ALLURE_ID_LABEL_NAME = "AS_ID";
```

Comment:
Should we support old labels?

Reply:
It's not much of overhead, and chances are someone has something like `[Label("AS_ID", "100500")]` in their legacy code (especially folks who previously used allure-python; `AS_ID` is still in use there). So overall I think yes, we should support it.